Sources/TruexAdRendererTarget/Sources.swift

```diff
@@ -0,0 +1,7 @@
+//
+//  Sources.swift
+//  TruexAdRenderer
+//
+//  Created by Andrew Tran on 10/23/24.
+//
+
```

Comment:
you need to test with an actual TAR import
create it, maybe query and log something interesting about it
e.g. it's version?